### PR TITLE
Initial project generation

### PR DESCRIPTION
**Pull request by hisaac**
This pull request adds the sample app generated by the React Native CLI to the repo, as well as a basic `bitrise.yml` file for demonstration purposes.

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    - deploy-to-bitrise-io@2: { }
+
+trigger_map:
+- pull_request_source_branch: "*"
+  workflow: primary
+  pull_request_target_branch: main
+
+meta:
+  bitrise.io:
+    stack: osx-xcode-13.2.x
```
Comment (ofalvai) [inline bitrise.yml]: Can we avoid hardcoding a stack here? I'm sure we'll forget to update this when new stacks are available, and one day builds will fail when the 13.2 stack is removed. What happens if we don't define a stack at all?

Comment (hisaac) [reply bitrise.yml]: Good call here. I'll discuss with the team how they're currently handling this when users create a build from the Getting Started page.

Comment (hisaac) [reply bitrise.yml]: Looks like the [iOS one is specifying a stack as well](https://github.com/bitrise-io/Bitrise-iOS-Cocoapods-Sample/blob/18fe2f5d9998930b3681cfd011f46ce624aba1b2/bitrise.yml#L12-L15), so maybe it's ok? I'll check with the team.

Comment (ofalvai) [reply bitrise.yml]: Whoops, my bad. Let me know if you find a quick and easy solution for this issue and I'll update the existing sample apps too

Comment (hisaac) [reply bitrise.yml]: haha, no worries. Wasn't pointing fingers, just wondering now if it effects the demo app builds. I'll find out and let you know. 👌

Comment (hisaac) [reply bitrise.yml]: This has been removed now 👍

**Review by ofalvai — APPROVED** [review]

**Pull request merged**